### ISSUE TITLE
No command-line parameters provided to antares-solver

### ISSUE THREAD

**Issue by flomnes** [AntaresSimulatorTeam/AntaREST#1594]
AntaREST v2.14.2 and v.14.3rc1

Desktop version

Ubuntu 20.04

See https://github.com/AntaresSimulatorTeam/AntaREST/issues/1574#issuecomment-1590827741

**Comment by laurent-laporte-pro** [comment]
Same issue in development environment:

```log
WARNING: Logging before InitGoogleLogging() is written to STDERR
W0619 15:23:42.465341 13988 environment.cc:187] Environment variable XPRESSDIR undefined.
W0619 15:23:42.465388 13988 environment.cc:249] NOT_FOUND: Could not find the Xpress shared library. Looked in: [/opt/xpressmp/lib/libxprs.so]. Please check environment variable XPRESSDIR
[2023-06-19 15:23:42][solver][0;31m[error][0m A study folder is required. Use '--help' for more information
[2023-06-19 15:23:42][solver][0;31m[fatal][0m Aborting now. (warning: no file log available)
[2023-06-19 15:23:42][solver][0;33m[warns][0m No log file available
```

**Comment by laurent-laporte-pro** [comment]
fixed in v2.14.3 #1597
